ios/react-native-xcode.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
```

Comment:
Why have you copied this file out of node_modules? Maybe add comments if you made any changes to it